### PR TITLE
Updating translations for search bar 

### PR DESCRIPTION
**Pull request by procrastinatio**
[Test link](https://sys-map.dev.bgdi.ch/preview/feat_bgdiinf_sb-2755_search_text/index.html)

### CODE REVIEW

**Review by pakb — COMMENTED** [review]
As it doubles down on the text present on desktop, we could get rid of this text oustide of the search bar to gain some screen space. What's your take on that?
image

**Comment by procrastinatio** [comment]
OK, for me.

**Comment by pakb** [comment]
Might be a bit hard to achieve, but would be great if the search bar was centered vertically. And the TESTSITE warning placed underneath, after having centered the searchbar.
You might need to rework the HTML hierarchy to achieve that, and use some of the helper classes found here : https://getbootstrap.com/docs/5.3/utilities/flex/#align-self

**Comment by procrastinatio** [comment]
Looks like it centered now. No idea why, though.

File: src/modules/menu/components/search/SearchBar.vue [hunk 1]
```diff
@@ -91,3 +91,17 @@ export default {
...
+.search-bar {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+
+
+
+</style>
```
Comment (pakb) [inline src/modules/menu/components/search/SearchBar.vue:107]: I'm not sure this is necessary, as the search bar is already centered by its parent (by HeaderWithSearchBar)

File: src/modules/menu/components/header/HeaderWithSearch.vue [hunk 2]
```diff
@@ -105,6 +105,7 @@ $animation-time: 0.5s;
     // On desktop we limit hte maximum size of the search bar just
     // to have a better look and feel.
     max-width: 800px;
+    position: relative;
```
Comment (pakb) [inline src/modules/menu/components/header/HeaderWithSearch.vue:108]: this too might be unnecessarry

**Review by pakb — COMMENTED** [review]
As I've used some bootstrap Flexbox magic to center all of this, it should be easier for you yes 😉

**Review by pakb — APPROVED** [review]

**Pull request merged**